beginner_source/onnx/export_simple_model_to_onnx_tutorial.py

```diff
@@ -7,12 +7,12 @@
 Export a PyTorch model to ONNX
 ==============================
 
-**Author**: `Thiago Crepaldi <https://github.com/thiagocrepaldi>`_
+**Author**: `Ti-Tai Wang <https://github.com/titaiwangms>`_
```

Comment:
We typically don't remove the original author even if they left the project. POC for reviews/updates are managed through [this issue](https://github.com/pytorch/tutorials/issues/1896) and labels.